O/openPMD_api/build_tarballs.jl

```diff
@@ -6,7 +6,7 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "openPMD_api"
-version = v"0.15.1"
+version = v"0.15.2"
 openpmi_api_version = "v.0.14.5" # This is really the `dev` branch after version 0.14.5
 
 # `v"1.6.3"` fails to build
```

Comment:
You may wish to 1.10 while at it (we just added this to libjulia_jll).

Reply:
@fingolfin `openPMD_api_jll` uses libcxxwrap; does this support Julia 1.10 already?

Reply:
Not yet, waiting for PR #5867 to be merged

Reply:
Never mind, I made #5883 which can be merged now, and it "commutes" with this PR here

Reply:
This was now done in a separate PR.